### PR TITLE
update to v5 structure

### PR DESCRIPTION
**Pull request by KennethBWSong**

### CODE REVIEW

File: web.config [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (SLdragon) [inline web.config:1]: Is this file necessary?

Comment (KennethBWSong) [reply web.config:1]: Confirmed with Fan: This file is necessary when deployed to azure app service to tell IIS that this project is a nodejs project.

**Comment by SLdragon** [comment]
Looks good, and merge it
